### PR TITLE
Use XHR directly instead of jquery post for plaintext request bodies.

### PR DESCRIPTION
**Pull request by nhnt11**
Untested. This should fix the issue Erik found where the POST tests are detecting unexpectedly mutated data. See https://github.com/mozilla/OpenWPM/pull/310#pullrequestreview-242449003

### CODE REVIEW

**Comment by eriktews** [comment]
I am a bit surprised by that error:

https://travis-ci.org/mozilla/OpenWPM/jobs/538062525#L2628

It looks like the request has never been sent or it is not properly stored by OpenWPM.

File: test/test_pages/post_request_ajax.html [hunk 1]
```diff
@@ -8,14 +8,18 @@
     <script type="text/javascript" src="shared/utils.js"></script>
     <script type="application/javascript">
 
+      function do_xhr(data) {
+        var xhr = new XMLHttpRequest;
```
Comment (nhnt11) [inline test/test_pages/post_request_ajax.html]: This should be `new XMLHttpRequest()`

Comment (eriktews) [reply test/test_pages/post_request_ajax.html]: That sounds much better. Can you update your branch so that the Travis-CI tests will run again?

Comment (englehardt) [reply test/test_pages/post_request_ajax.html]: Fixed this and triggered a re-run of the tests.

**Comment by englehardt** [comment]
This was fixed in #316

**Pull request closed without merging**